x-pack/plugins/monitoring/public/alerts/alert_form.test.tsx

```diff
@@ -154,8 +154,8 @@ describe('alert_form', () => {
 
     it('renders alert name', async () => {
       const alertNameField = wrapper.find('[data-test-subj="ruleNameInput"]');
-      expect(alertNameField.exists()).toBeTruthy();
       expect(alertNameField.first().prop('value')).toBe('test');
+      expect(alertNameField.exists()).toBeTruthy();
```

Comment:
Could you help me understand the meaning of this reordering of assertions?

Reply:
It's a leftover after backing out changes of inputs optimisation try in alert flyout.

Reply:
@Dmitriynj think since this change is not relevant to this PR, it should be cleaned up

Reply:
Done